k8s/scripts/run_docker.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -x  # show commands being ran
+set -e  # error if anything fails
+set -o pipefail # error on pipe fails too
+
+current_version="latest"
```

Comment:
does GCR allow us to overwrite the latest image every time we push?

Reply:
It'll overwrite the latest image. But it'll keep all previous images as well.